.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        django_version: ["2.2"]
+        python_version: ["3.6", "3.7", "3.8"]
+        variant: [alpine, slim]
+        include:
+          - variant: alpine
+            pg_major: 12
+            pg_version: 12.2-r0
+          - variant: slim
+            pg_major: 10
+            pg_version: 10.13-1.pgdg100+1
+    env:
+      CI: 1
+      DOCKER_BUILDKIT: 1
+      VERSION: ${{ matrix.django_version }}
+      PYTHON_VERSION: ${{ matrix.python_version }}
+      PG_MAJOR: ${{ matrix.pg_major }}
+      PG_VERSION: ${{ matrix.pg_version }}
+      VARIANT: ${{ matrix.variant }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: ./scripts/cibuild
+
+      - run: ./scripts/cipublish
+        if: github.ref == 'refs/heads/master'
+        env:
+          QUAY_USER: ${{ secrets.QUAY_IMAGES_USER }}
+          QUAY_PASSWORD: ${{ secrets.QUAY_IMAGES_PASSWORD }}
```

Comment:
I have the credentials for our `azavea+images` user set as an organizational secret. I've granted access to those secrets to this repo and `azavea/docker-tippecanoe` (which we can update to use this secret if we end up using it here). I namespaced it with `IMAGES` since we have a few different machine users that we use for different application groups.

I think we should use organizational secrets in situations like this to make it easier to revoke / roll credentials that are used in multiple repositories.